docs/snippets/prerequisites.md

```diff
@@ -2,17 +2,17 @@
 
 Before installing Knative, you must meet the following prerequisites:
 
-- **For prototyping purposes**, Knative will work on most local deployments of Kubernetes. For example, you can use a local, one-node cluster that has 2 CPU and 4GB of memory.
+- **For prototyping purposes**, Knative works on most local deployments of Kubernetes. For example, you can use a local, one-node cluster that has 2&nbsp;CPUs and 4&nbsp;GB of memory.
```

Comment:
same as below

Reply:
[Google Style](https://developers.google.com/style/units-of-measure#spaces-in-units-of-measurement) asks for non-breaking spaces.